src/mixed-line-bar-chart/__tests__/mixed-chart.test.tsx

```diff
@@ -928,8 +928,6 @@ describe('Details popover', () => {
 
     wrapper.findChart()!.fireEvent(new MouseEvent('mousedown', { bubbles: true }));
 
-    expect(wrapper.findDetailPopover()!.findDismissButton()!.getElement()).toHaveFocus();
```

Comment:
this is now tested in integ tests (see above) as I was unable to get it to work stably here (and testing such interactions in unit tests anyway can be quite difficult/problematic)